<resources>
    <!-- Network errors -->
    <string name="error_no_connection">No connection</string>
    <string name="error_unauthorized">Error: Unauthorized</string>
    <string name="error_forbidden">Error: Forbidden</string>
    <string name="error_not_found">Error: Not found</string>
    <string name="error_rate_limit">Error: You have reached the API rate limit. Try again later</string>
    <string name="error_abuse_limit">Error: GitHub has flagged you for API abuse</string>
    <string name="error_max_login_attempts">Error: Maximum number of login attempts reached. Try again later</string>
    <string name="error_unprocessable">Error: There was an error processing your request</string>
    <string name="error_bad_credentials">Error bad credentials</string>
    <string name="error_not_allowed">Error: Not allowed</string>
    <string name="error_bad_request">Error: Bad request</string>
    <string name="error_empty_repository">Error: Empty repository</string>
    <string name="error_unknown">Unknown error</string>
</resources>
